Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
+
+    [root@~]#cat /var/lib/glusterd/vols/tv1/info  | grep ^op-version
+
+
+If this is not done automatically set op-version as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
+
+### Understanding op-version
+
+The op-version for a given X.Y.Z release will be an integer XYZ, with
```

Comment:
I have looked in libglusterfs/src/globals.h and updated this info.
/* Gluster versions - OP-VERSION mapping
 *
 * 3.3.x                - 1
 * 3.4.x                - 2
 * 3.5.0                - 3
 * 3.5.1                - 30501
 * 3.6.0                - 30600
 * 3.7.0                - 30700
 * 3.7.1                - 30701
 * 3.7.2                - 30702
 *
 * Starting with Gluster v3.6, the op-version will be multi-digit integer values
 * based on the Glusterfs version, instead of a simply incrementing integer
 * value. The op-version for a given X.Y.Z release will be an integer XYZ, with
 * Y and Z 2 digit always 2 digits wide and padded with 0 when needed. This
 * should allow for some gaps between two Y releases for backports of features
 * in Z releases.
 */


<<to have a metric for all our active releases with the op-version details and then this metric can be linked to the upgrade guide.

hmm... just feel its getting overly complicated.
Lets keep this document simple, covering  basic aspects of op-version.